lib/rspec/core.rb

```diff
@@ -100,23 +100,7 @@ def self.configure
   # The example being executed.
   #
   # The primary audience for this method is library authors who need access
-  # to the example currently being executed and also want to support RSpec 2.
-  #
-  # @example support for RSpec version 2
-  #
-  #     RSpec.configure do |c|
-  #       # context.example is deprecated, but RSpec.current_example is not
-  #       # available until RSpec 3.0.
-  #       fetch_current_example = RSpec.respond_to?(:current_example) ?
-  #         proc { RSpec.current_example } : proc { |context| context.example }
-  #
-  #       c.before(:example) do
-  #         example = fetch_current_example.call(self)
-  #
-  #         # ...
-  #       end
-  #     end
-  #
+  # to the example currently being executed.
```

Comment:
I'm on the fence about this change.

Reply:
`RSpec.current_example` is available for quite a while now.
If there are libraries out there that aim to support both RSpec 2 and RSpec 3+, they for sure have this code built-in already.
I can't think of a new library that would want to support RSpec 2.
Keeping this documentation would only help the authors of those libraries.

Does it make sense to keep it around?

Reply:
Ok convinced.